config/services.php

```diff
@@ -31,4 +31,10 @@
         'region' => env('AWS_DEFAULT_REGION', 'us-east-1'),
     ],
 
+    'google' => [
+        'client_id' => '666962700530-24bskaf6na5du7t5p03lslvgb3fk8u80.apps.googleusercontent.com',
```

Comment:
Please put these values in the env file, and use the `env` helper function to get the value.